internal/venice-common/src/main/java/com/linkedin/venice/meta/StoreInfo.java

```diff
@@ -143,6 +144,11 @@ public static StoreInfo fromStore(Store store) {
    */
   private boolean chunkingEnabled = false;
 
+  /**
+   * Whether the replication metadata chunking is enabled for Active/Active replication enabled store.
+   */
+  private boolean replicationMetadataChunkingEnabled = false;
```

Comment:
Let's shorten `replicationMetadataChunkingEnabled` to `rmdChunkingEnabled` to save some space. We've used `rmd` as the short form for replication metadata everywhere in code now